src/providers/SignatureProvider.tsx

```diff
@@ -10,7 +11,47 @@ interface InputElementOptions {
 
 const createSignatureProviderState = () => {
     const [imgUrl, setImgUrl] = useState<string>();
+
     const [name, setName] = useState<InputElementOptions>({
+        type: "name",
+        textValue: "",
+        isBold: false,
+        isItalics: false,
+        style: { fontWeight: "normal", fontStyle: "normal" },
+    });
+
+    const [email, setEmail] = useState<InputElementOptions>({
+        type: "email",
+        textValue: "",
+        isBold: false,
+        isItalics: false,
+        style: { fontWeight: "normal", fontStyle: "normal" },
+    });
```

Comment:
🤔 There's a ton of duplicate code here ... Could we create a helper function that initializes the default value ? 

```suggestion
    const [email, setEmail] = useState<InputElementOptions>(getDefaultValue("email"));
```